verification.py

```diff
@@ -1871,7 +1884,7 @@ class RecombinationMutationTest(Test):
     """
 
     def verify_recombination(
-        self, name, sample_size, Ne, r, m, L, model, growth_rate=0
+        self, name, sample_size, Ne, r, m, L, p, model, growth_rate=0
```

Comment:
Probably better to say ``ploidy`` here rather than p - I'm a bit hazy on what these single letter parameters mean and they're usually best avoided.